typed/remove.go

```diff
@@ -46,8 +48,10 @@ func (w *removingWalker) doScalar(t *schema.Scalar) ValidationErrors {
 func (w *removingWalker) doList(t *schema.List) (errs ValidationErrors) {
 	l := w.value.AsListUsing(w.allocator)
 	defer w.allocator.Free(l)
-	// If list is null, empty, or atomic just return
-	if l == nil || l.Length() == 0 || t.ElementRelationship == schema.Atomic {
```

Comment:
@jennybuckley Do you remember why we had the atomic check here? Would it be a problem to remove it?